testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/permissionmappers/ConstantPermissionMapperTestCase.java

```diff
@@ -276,10 +275,8 @@ private void setParam(List<NameValuePair> nvps, final String paramName, String p
     private static WebArchive createWar(final String sd) {
         return ShrinkWrap.create(WebArchive.class, sd + ".war").addClasses(CheckIdentityPermissionServlet.class)
                 .addAsWebInfResource(Utils.getJBossWebXmlAsset(sd), "jboss-web.xml")
-                .addAsManifestResource(PermissionUtils.createPermissionsXmlAsset(new ElytronPermission("*"),
-                        // FIXME remove this FilePermission once the ELY-1055 is fixed
-                        new FilePermission(System.getProperty("jboss.inst") + "/-", "read") // ELY-1055 workaround
-                ), "permissions.xml")
+                .addAsManifestResource(
+                        PermissionUtils.createPermissionsXmlAsset(new ElytronPermission("*")), "permissions.xml")
```

Comment:
I believe using accurate permissions which are required would be better than using allPermission.

Reply:
will remove from this PR